Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM php:7.2.25-fpm
```

Comment:
overstappen naar php 7.3? of zelfs 7.4? 7.3 draaien we nu ook op de server geloof ik

Reply:
ik zal de 7.3 versie ook klaarmaken op de test server

Reply:
7.4 versie kan ik makkelijk testen, maar zullen we dit uitstellen na de docker prod update? ik wil eerst graag zien hoe het in een docker omgeving draait met dezelfde opzet

Reply:
Ja goed idee!

Reply:
net ff snel getest, ik zie dat niet alle packages goed geinstalleerd worden op 7.4, dus dit is een item voor later

Reply:
was toch makkelijker dan verwacht, heb er een nieuwe branch voor aangemaakt, maar php 7.4 werkt, alleen moeten we eerst ook travis aanpassen, dus update word postponed